helm/spark-yotpo/spark-with-atlas.yaml

```diff
@@ -0,0 +1,6 @@
+useAtlas:
+  atlas.kafka.zookeeper.connect: ${ATLAS_ZOOKEEPER_CONNECT}
```

Comment:
the first 2 lines are pushed to a file called `/spark/conf/atlas-application.properties` we need to check how to do it in k8s, im thinking maybe to do it inside the docker

Reply:
could it be that the original atlas-application.properties is just pushing these properties to the spark conf?